### ISSUE TITLE
Support for 4.0?

### ISSUE THREAD

**Issue by tomswinkels** [graphaware/neo4j-php-client#170]
Is this version supported for version 4.0?

**Comment by DeltaBravo** [comment]
No, not currently: https://github.com/graphaware/neo4j-php-client/issues/166#issuecomment-568457302

**Comment by Ileca** [comment]
Is there a known alternative to be able to use PHP with 4.0.1? If not, the other thread says no support for recent 3.5.x but what x exactly is supported?

**Comment by DeltaBravo** [comment]
There is currently no alternative. This is, and has been for the past 5 years, the best Neo4j PHP driver. The other drivers are in an even worse state. In my opinion Web developers are not important to Neo4j--they want enterprise level customers.

I am running 3.5.14 without any issues aside from deprecation warnings when running tests.

**Comment by tomswinkels** [comment]
Look at https://github.com/stefanak-michal/Bolt maybe later there is support for 4.0.

**Comment by DeltaBravo** [comment]
That project sites the same issue as this project--Neo4j has not released documentation on the protocol. Neo4j seems to be focused on enterprise level customers and could not care less about common use cases such as Web development. They seem to be running things using the Oracle business model.

The only way I see a Neo4j PHP driver is if someone spends the time and money to reverse engineer / packet sniff the bolt protocol. Even then we are still missing critical features such as multi-tenancy which are only available in the ridiculously priced enterprise edition.

**Comment by Ileca** [comment]
This situation boggles my mind. They dropped support while preventing people from making adjustments. It's been seven months since they drop! I started learning how to use neo4j and now I am ready I can't even be up to date (it also means it's not reliable)...
I see 500k downloads for this composer package. I wonder how those who are using it with their website feel about being abandoned just because they can't be milked for money. At least, Oracle-Mysql is well integrated to PHP.

Have you guys thought about spamming them so they revise their priorities?
https://community.neo4j.com/t/can-we-use-neo4j-4-0-with-php/15754 is the only thread I see about that on their forum.
There is this issue on their github: https://github.com/neo4j/neo4j/issues/12361
